### ISSUE TITLE
[Bug]: Configuring a mapped folder to a network share no longer works in 2024H2

### ISSUE THREAD

**Issue by elisimpson** [microsoft/Windows-Sandbox-Utilities#63]
### Windows Sandbox version

latest version

### Windows build number

Windows 11 2024H2

### Steps to reproduce the bug

1. Use a .wsb config file to map a folder to a network share:
```
<Configuration>
    <MappedFolders>
        <MappedFolder>
            <HostFolder>\\foo\bar\baz</HostFolder>
            <SandboxFolder>C:\Test</SandboxFolder>
        </MappedFolder>
    </MappedFolders>
</Configuration>
```

2. Launch Sandbox with the config file will result in a error: `Error 0x8007052e. The user name or password is incorrect.`

The exact same process on a 2023H2 install will work. Something changed with 2024H2 which breaks Sandbox mapped folder functionality, specifically the login credentials. To be clear, the correct login credentials are saved on the host machine and the network resource is available.

### Expected result

Mapped folder to a network share should work in 2024H2 the same as it did in 2023H2

### Actual result

2024H2 will fail

**Comment by kvega005** [comment]
Due to security concerns, we cannot share folders from a network share to the Virtual Environment. This would require the Sandbox VM to access network to which it is not authorize. The only option here is to save the login credentials and access the network share from within the Windows Sandbox environment.

**Comment by Conduitry** [comment]
How did this work in 2023H2? It also doesn't seem like the sandboxed system would have to have broad access to these networks any more than it would need to have broad access to regular disks containing folders that have been mounted inside the sandboxed system.

If I wanted to use your proposed solution, what would I do? If I'm fine manually authenticating with specific shared folders on specific devices, how do I get at any of these devices on the host machine's network? Going into Windows Explorer > Network in the sandboxed machine, I'm not seeing any of the devices from the host network.

**Comment by Zoddo** [comment]
I agree with the above comment. This is not a feature request, but a regression bug report. Mounting an UNC folder inside the sandbox has always worked on Windows 10/11 (before 24H2), even if the sandbox's network access was disabled in the `.wsb` file.

Additionally, Accessing a network path from the sandbox (with network access enabled) doesn't work for me either.

@kvega005 is it possible to reopen this issue?
